### PR TITLE
Music: AI drums tweak

### PR DESCRIPTION
**Pull request by breville**
A small tweak to the AI drums UI which moves the temperature slider above the `Generate` button.

This is a more natural arrangement, since a user would adjust the slider first, and then move down to the button.

### After

Screenshot 2024-10-07 at 9 47 29 PM

### CODE REVIEW

File: apps/src/music/views/PatternAiPanel.tsx [hunk 1]
```diff
@@ -434,6 +426,15 @@ const PatternAiPanel: React.FunctionComponent<PatternAiPanelProps> = ({
                 </div>
               </div>
             )}
+            <Button
+              ariaLabel={musicI18n.generate()}
+              text={musicI18n.generate()}
+              onClick={handleAiClick}
+              disabled={generateState === 'generating'}
```
Comment (levadadenys) [inline apps/src/music/views/PatternAiPanel.tsx:433]: super nit: I see that it's not part of this PR, but when I've seen this line I thought about asking. Don't we want to use `isPending` state of `Button` here instead of `disabled`?  The only drawback here is that we'll need modify `handleAiClick` callback so that it will do nothing when `generateState === 'generating'`, plus use `isPending` instead of `disabled` prop

**Review by levadadenys — APPROVED** [review]

File: apps/src/music/views/PatternAiPanel.tsx (hunk 1 above)
Comment (breville) [reply apps/src/music/views/PatternAiPanel.tsx:433]: Good question, and I'm not yet sure.  We have some significant design revisions to come for this UI, and we'll probably make use of a fairly large part of the panel to show that the generating process is underway.  If the button were the only place to show that we're waiting, `isPending` would make sense, but if we are using a separate area to show this, then the button having a spinner might be distracting.
